packages/expo-dev-launcher/ios/EXDevLauncherController.m

```diff
@@ -187,7 +187,15 @@ - (BOOL)onDeepLink:(NSURL *)url options:(NSDictionary *)options
   NSURL *appUrl = [EXDevLauncherURLHelper getAppURLFromDevLauncherURL:url];
   if (appUrl) {
     [self loadApp:appUrl onSuccess:nil onError:^(NSError *error) {
-      NSLog(@"%@", error.description);
+      __weak typeof(self) weakSelf = self;
```

Comment:
There's no need to do the weak-strong-self dance here since the block isn't saved as a property anywhere.

Reply:
This block is passed to the HTTP client so I will be stored somewhere. We could theoretically give up with this weak conversion because the `EXDevLauncherController` is a singleton. However, I think we should leave it here to be extra safe.